packages/ibm-products/src/components/Datagrid/useDisableSelectRows.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 5724-Q36
+ * (c) Copyright IBM Corp. 2020
```

Comment:
I think, we need to append the year of recent update to this line
 * (c) Copyright IBM Corp. 2020
```suggestion
 * (c) Copyright IBM Corp. 2020 - 2024
```